chat-components/ui/src/main/java/com/pubnub/components/chat/util/AutoSizeText.kt

```diff
@@ -145,7 +145,7 @@ private fun BoxWithConstraintsScope.shouldShrink(
         true,
         TextOverflow.Clip,
         LocalDensity.current,
-        LocalFontLoader.current,
+        LocalFontFamilyResolver.current,
```

Comment:
@kazydek `LocalFontFamilyResolver` instead of `LocalFontLoader` (new Compose API)

Reply:
@Chesteer89 Thx, it wasn't mentioned in the docs so no update was required.